.github/workflows/test-indexer-rpm.yml

```diff
@@ -0,0 +1,32 @@
+name: Test the preserving of security config files upon upgrade - Wazuh indexer - RPM
+on:
+  pull_request:
+    paths:
+      - 'stack/indexer/rpm/wazuh-indexer.spec'
+  workflow_dispatch:
+
+jobs:
+  Test-security-config-files-preservation-RPM:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the latest stable Wazuh version (all components)
+        run: echo "LATEST_STABLE_VERSION=$(jq -r 'map(select(.prerelease == false and .draft == false)) | .[] | .tag_name' <<< $(curl --silent https://api.github.com/repos/wazuh/wazuh/releases) | sed "s|v||g" | sort -rV | head -n 1)" >> $GITHUB_ENV
+
+      - name: Get the major and minor of the release version
+        run: echo "MAJOR_MINOR=$(echo $LATEST_STABLE_VERSION | cut -d '.' -f1-2 | sed "s|\.||")" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v3
+      - name: Build the Wazuh indexer package and set environment variable
+        working-directory: ./stack/indexer/rpm
+        run: |
+          sudo ./build_package.sh
+          echo "PACKAGE_NAME=$(ls ./output | grep .rpm | head -n 1)" >> $GITHUB_ENV
+
+      - name: Move the built package
+        working-directory: ./stack/indexer/rpm
+        run: |
+          mkdir $GITHUB_WORKSPACE/packages
+          sudo cp ./output/$PACKAGE_NAME $GITHUB_WORKSPACE/packages/$PACKAGE_NAME
+
+      - name: Launch docker
+        run: sudo docker run -v $GITHUB_WORKSPACE/.github/actions/upgrade-indexer/:/tests -v $GITHUB_WORKSPACE/packages/:/packages centos:centos7 bash /tests/upgrade-indexer.sh /packages/$PACKAGE_NAME $MAJOR_MINOR
```

Comment:
Add a new line at end of file

Reply:
Done in https://github.com/wazuh/wazuh-packages/commit/f72e2418b5b5ff9c48c5a25bee1fc380bdac07cf